code/more-dom-manipulation/JavaScript.js

```diff
@@ -0,0 +1,61 @@
+
+
+
+//Create a new img element and set its src attribute to a picture of you. Put that new img element in the div with the class of "profilePicture"
+const image = document.querySelector('.profilePicture');
+console.log(image);
+
+//Change the font-family of the page to "Arial, sans-serif"
+const content = document.querySelector('body');
+content.style.fontFamily ='Arial,sans-serif';
+
+//Change each li's text color to "rebeccapurple"
+const listitem = document.querySelector('li');
```

Comment:
You want to select all the list items, so instead of `querySelector()` use `querySelectorAll()`